src/main/jhipster/src/main/webapp/app/shared/list-creation/model/germplasm-list.ts

```diff
@@ -16,6 +16,8 @@ export class GermplasmListEntry extends ListEntry {
         public entryCode?: string,
         public seedSource?: string,
         public groupName?: string,
+        // {variableId: GermplasmListObservationDto}
```

Comment:
is this commented for any reason? or whould we remove the comment? 
@nahuel-soldevilla

Reply:
I replaced it with a proper javadoc style comment